deployables/extension/src/panel/integrations/safe/isSafeAccount.ts

```diff
@@ -0,0 +1,13 @@
+import type { ChainId } from '@zodiac/chains'
+import { initSafeApiKit } from '@zodiac/safe'
+
+export const isSafeAccount = async (chainId: ChainId, safeAddress: string) => {
```

Comment:
```suggestion
export const isSafeAccount = async (chainId: ChainId, safeAddress: HexAddress) => {
```

Is that a fair assumption, so that we can stay type-safe around addresses as much as possible?